django/db/backends/sqlite3/operations.py

```diff
@@ -103,6 +103,41 @@ def drop_foreignkey_sql(self):
     def pk_default_value(self):
         return "NULL"
 
+    def _quote_params_for_last_executed_query(self, params):
+        """
+        Only for last_executed_query! Don't use this to execute SQL queries!
+        """
+        sql = 'SELECT ' + ', '.join(['QUOTE(?)'] * len(params))
```

Comment:
Do we have sufficient test coverage? In particular, I'm looking for a test which verifies this "QUOTE(?)" bit.